ansible-ipi-install/roles/node-prep/tasks/40_bridge.yml

```diff
@@ -68,7 +68,15 @@
     mode: u=rw,g=r,o=r
   become: yes
 
-- name: Get DHCP for baremetal bridge
+- name: Set DNS priority for baremetal interface
   shell: |
-    /usr/sbin/ifdown {{ pub_nic }}; /usr/bin/pkill dhclient ; /usr/sbin/ifdown baremetal ; /usr/sbin/ifup {{ pub_nic }}; /usr/sbin/ifup baremetal
+    /usr/bin/nmcli con mod baremetal ipv4.dns-priority 10
```

Comment:
I can probably remove this from here. It needs to be in our lab prerequisite instructions anyway.

Early on when you run the task to verify the DNS records, it will fail in our environment if the `/etc/resolv.conf` file has the nameserver entries from the public interface listed above our local dnsmasq server. So we need for our `baremetal` bridge to be configured with the higher (er, lower) `dns-priority` to ensure that our local nameserver gets listed first.

Reply:
Ok cool - yeah I vote to remove as its specific for your env